src/param.F

```diff
@@ -220,12 +221,13 @@ subroutine param(namelist_input,                                  &
          read(20,nml=nssl2mom_params)
       ENDIF
       close(unit=20)
-      !$acc enter data copyin(dx,dy,dz,wbc,ebc,sbc,nbc)
       
 
 #ifdef MPI
       endif  myid0
+#endif
 
+#ifdef MPI
       call MPI_BCAST(dx    ,1,MPI_REAL,0,MPI_COMM_WORLD,ierr)
```

Comment:
shall we change these calls to the new `bcast_int` or `bcast_real` wrappers as well?

Reply:
We will eventually replace all of these MPI_BCAST calls.    This will be a decent amount of work as there will be a need to add the corresponding $acc declare create() calls for all of the variables.  I thought that this was beyond the scope for this particular pull request.

Reply:
Thanks.